api/v1/controllers/categories.js

```diff
@@ -0,0 +1,53 @@
+import { toInt } from '../helpers/functions';
```

Comment:
Remember that you're to prefer using `parseInt` directly over using the helper function

Reply:
Okay I'll change it